src/index.html

```diff
@@ -5,8 +5,317 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body id="#home">
+    <header class="header">
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="#"
+          class="header__icon"
+        >
+      </a>
+      <nav class="nav">
```

Comment:
```suggestion
      <a href="#home">
        <img
          src="./images/logo.png"
          alt="#"
          class="header__icon"
        >
      </a>
      
      <nav class="nav">
```
add blanks between multiline elements/blocks

Reply:
isn't fixed all over the page. Please, check inside your articles

Reply:
is still not fixed all over the page. Please, check inside your articles. Add an empty lines between multiline siblings